### ISSUE TITLE
Panel will not connect to daemon, but daemon will connect to panel

### ISSUE THREAD

**Issue by chaoschao11** [ccarney16/pterodactyl-containers#109]
For some reason the daemon can connect with and pull panel info to create servers but the panel is unable to then connect back to the daemon. This leads to a situation where the server isn't actually accessible in the panel. We have narrowed it down to a networking issue of some sort but error logs are not getting us very far. As a side note we allocated the internal docker ip range but that did not work either. 

TL;DR For some reason the web socket isn't connecting but the panel is able to connect to the daemon and create new servers

Photo attached for reference:

![esrsrsrsrse](https://user-images.githubusercontent.com/1449544/116487136-45b75600-a85d-11eb-8a47-180124184545.png)

**Comment by ccarney16** [comment]
Have you setup pterodactyl before? You need to have both the panel and daemon open for client connections, meaning the web browser must see the daemon also.

**Comment by chaoschao11** [comment]
All good we got it figured out! Thanks for the help.
